src/main/java/com/robothand/highqualitybot/Config.java

```diff
@@ -0,0 +1,47 @@
+package com.robothand.highqualitybot;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Hashtable;
+import java.util.Scanner;
+
+/**
+ * Config.java
+ *
+ * Reads in properties from a specified configuration file, and provides access to the values.
+ */
+public class Config {
+    private final File configFile;
+
+    public String TOKEN;
+    public String PREFIX;
+
+
+    public Config(String path) throws FileNotFoundException {
+        configFile = new File(path);
+
+        Scanner s = new Scanner(configFile);
+
+        while (s.hasNextLine()) {
+            String[] split = s.nextLine().split("=");
+            String key, value;
+
+            if (split.length < 2 || split[0].trim().startsWith("#")) {
+                continue;
+            }
+
+            key = split[0].trim();
+            value = split[1].trim();
+
+            // place the property if it is known
+            switch (key) {
```

Comment:
Add a default case with a comment saying something along the lines of `unknown property`